frontend/src/components/annotation/text_annotation/text_annotation_core.vue

```diff
@@ -373,7 +373,7 @@ export default Vue.extend({
   },
   computed: {
     real_container_width: function() {
-      if (this.container_width) return this.container_width - 100
+      if (this.container_width) return this.container_width - 30
```

Comment:
Changed `this.container_width - 100` to `this.container_width - 30` because `100px` padding on the right side looked very excessive and didn't seem like UI real estate was used properly.